src/pages/pricing/index.tsx

```diff
@@ -12,10 +13,17 @@ import emailIsValid from 'utils/email-is-valid'
 export const getServerSideProps: GetServerSideProps = async function ({res}) {
   res.setHeader('Cache-Control', 's-maxage=1, stale-while-revalidate')
 
-  if (!process.env.NEXT_PUBLIC_STRIPE_ANNUAL_PRICE_ID)
-    throw new Error('no annual price to load 😭')
+  const {data: pricingData} = await axios.get(
+    `${process.env.NEXT_PUBLIC_AUTH_DOMAIN}/api/v1/pricing`,
+  )
+  // TODO: Is finding the annual plan by name reliable?
+  const proYearly: {price_id: string} | undefined = pricingData.plans.find(
+    (plan: any) => plan.name === 'Pro Yearly',
+  )
```

Comment:
Not sure how standard the naming is

Reply:
no, using lodash syntax:

```js
find(plans, {interval: 'year', type: 'individual', tier: 'pro'})
```

tested here just to verify: https://codesandbox.io/s/beautiful-lamport-4twub?file=/src/App.js

Reply:
I went ahead and updated this to just load from the egghead api